### PR TITLE
Makefile Documentation

### PR DESCRIPTION
**Pull request by ayusharma**
Ref #issue483

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,6 +13,26 @@ TEMPDIR := $(shell mktemp -d)
...
+	@echo "  migrate                     run the kinto migrations"
+	@echo "  tests-once                  only run the tests once with the default python interpreter"
+	@echo "  flake8                      run the flake8 linter"
+	@echo "  tests                       run all the tests with all the supported python interpreters (same as travis)"
+	@echo "  clean                       remove *.pyc files and __pycache__ directory"
+	@echo "  distclean                   remove *.egg-info files and *.egg, build and dist directories"
+	@echo "  maintainer-clean            remove the .tox and the .venv directories"
+	@echo "  loadtest-check-tutorial     load test the using tutorial"
+	@echo "  loadtest-check-simulation   load test using a simulation"
+	@echo "  docs                        builds the docs"
```
Comment (leplatrem) [inline Makefile]: nit: use infinitive everywhere for consistency

File: Makefile [hunk 2]
```diff
@@ -13,6 +13,26 @@ TEMPDIR := $(shell mktemp -d)
...
+	@echo "Check the Makefile to know exactly what each target is doing."
+
 all: install
```
Comment (leplatrem) [inline Makefile]: We may want to set `help` as the default target ?

File: Makefile [hunk 3]
```diff
@@ -13,6 +13,26 @@ TEMPDIR := $(shell mktemp -d)
 
 OBJECTS = .venv .coverage
 
+help:
+	@echo "Please use 'make <target>' where <target> is one of"
+	@echo "  install                     install dependencies and prepare environment"
+	@echo "  install-monitoring          enable monitoring feature using NewRelic"
```
Comment (leplatrem) [inline Makefile]: monitoring features like StatsD and Newrelic

File: Makefile (hunk 2 above)
Comment (magopian) [inline Makefile]: The very first target in a Makefile is the default target, so `help` should be the default one here.

File: Makefile (hunk 1 above)
Comment (ayusharma) [inline Makefile]: Pls take a look.. https://public.etherpad-mozilla.org/p/kinto-makefile-documentation . I tried to made description infinitive.

Comment (leplatrem) [inline Makefile]: I just said that because of the trailing *s* in *builds*.

Adding leading `to ` seems superfluous

**Comment by leplatrem** [comment]
Thanks for this contribution! This is ready to merge!

**Pull request merged**